### PR TITLE
Add supported DDL operations

### PR DESCRIPTION
**Pull request by flowbehappy**

### CODE REVIEW

**Comment by breezewish** [comment]
Hold until https://github.com/pingcap-inc/tidb2dw/pull/31 is merged

File: README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ export AWS_SESSION_TOKEN=<SESSION_TOKEN>  # Optional
...
-> Restart incremental replication without manually runing DDL on target table may cause data loss.
+## Supported DDL Operations
+
+* Drop column
+* Alter (rename) column
+* Alter column' (default) value
+* Drop/truncate tables
+* Change column type
+* Multi schema change
+* Add table partition
```
Comment (Lloyd-Pottiger) [inline README.md:48]:
```suggestion
- Add column
- Drop column
- Alter column rename
- Alter column modify column type
- Drop table
- Truncate table
```

> **Note**
> 1. Snowflake does not support partition table, tidb2dw will view table with multiple partitions as ordinary table.
> 2. Snowflake does not support update column default value, be careful.
> 3. The type mapping from TiDB to Snowflake is defined [here](./snowsql/types.go)

Comment (Lloyd-Pottiger) [reply README.md:48]: Snowflake has a lot of limitations on modifying column type, should be careful. refer to https://docs.snowflake.com/en/sql-reference/sql/alter-table-column

Comment (Lloyd-Pottiger) [reply README.md:48]: 4. Should execute any DML before DDL or will report error.

Comment (Lloyd-Pottiger) [reply README.md:48]: ## Supported DDL Operations

All DDL which will change the schema of table are supported (except index related), including:

- Add column
- Drop column
- Rename column
- Modify column type
- Drop table
- Truncate table

> **Note**
> 1. Snowflake does not support partition table, tidb2dw will view table with multiple partitions as ordinary table.
> 2. Snowflake has a lot of limitations on modifying column type, like Snowflake does not support update column default value, refer to [Snowflake Docs](https://docs.snowflake.com/en/sql-reference/sql/alter-table-column).
> 3. The type mapping from TiDB to Snowflake is defined [here](./snowsql/types.go).
> 4. Should execute any DML before DDL or will report error.

**Pull request merged**